app/controllers/android/httprequest.js

```diff
@@ -0,0 +1,31 @@
+var VolleyPackage = require("com.android.volley.*");
+var JSONPackage = require("org.json.*");
+var activity = new Activity(Ti.Android.currentActivity);
+
+function startRequest() {
+
+    var queue = VolleyPackage.toolbox.Volley.newRequestQueue(activity);
+    var url ="https://appcelerator.com";
+
+    var request = new VolleyPackage.toolbox.JsonObjectRequest(VolleyPackage.Request.Method.GET, url, 
```

Comment:
So I think the reason this isn't working is because when I added wildcard package support, I didn't "chain" the packages together. So you can do wildcard requires, but only to access types directly under that package, not to access sub-packages like you're doing here.

If you do a ```var VooleyToolbox = require('com.android.volley.toolobox.*');``` at top and use that, it should work.

I could easily add hanging sub-packages off the parent in JS code, but my concern here is how we'd handle this in generating the list of types used at build-time to limit our metabase/wrapper generation. Wildcards can be used, but I'd argue are wasteful generally in that they end up pulling in all the types in a given package. If we extend that to also include all types in every sub-package as well, that'd be **very** wasteful.